Source/Scene/Cesium3DTileStyle.js

```diff
@@ -246,13 +246,24 @@ define([
                 return this._show;
             },
             set : function(value) {
+                if (typeof value === 'boolean') {
+                    this._show = new Expression(String(value));
```

Comment:
I don't think it hurts to pass the `defines` in even if they are no longer relevant to the new style, at the very least it does cut down on the code duplication. But if you would prefer that to be a different PR that is also ok with me.